Islandora-CLAW/controlled_access_terms
# Re-export so you can install jsonapi

Resolves https://github.com/Islandora-CLAW/CLAW/issues/1054

The culprit was that the 'Broader' entity reference had its storage set up for Content, instead of other geo_location taxonomy terms.  There's some other stuff there that snuck in from the export.  Some of it is just things getting shuffled around, but some lang stuff is in there, too.

## Testing instructions
- Spin up a claw-playbook
- `composer require drupal/jsonapi`
- `drush en jsonapi`

You should see the following explosion:
```
vagrant@claw:/var/www/html/drupal/web$ drush en jsonapi
TypeError: Argument 2 passed to Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}() must be an instance of Drupal\jsonapi\ResourceType\ResourceType, null given in /var/www/html/drupal/web/modules/contrib/jsonapi/src/Routing/Routes.php on line 435 #0 [internal function]: Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}(false, NULL)
#1 /var/www/html/drupal/web/modules/contrib/jsonapi/src/Routing/Routes.php(437): array_reduce(Array, Object(Closure), false)
#2 /var/www/html/drupal/web/modules/contrib/jsonapi/src/Routing/Routes.php(340): Drupal\jsonapi\Routing\Routes::hasNonInternalTargetResourceTypes(Array)
#3 /var/www/html/drupal/web/modules/contrib/jsonapi/src/Routing/Routes.php(179): Drupal\jsonapi\Routing\Routes::getIndividualRoutesForResourceType(Object(Drupal\jsonapi\ResourceType\ResourceType))
#4 /var/www/html/drupal/web/modules/contrib/jsonapi/src/Routing/Routes.php(112): Drupal\jsonapi\Routing\Routes::getRoutesForResourceType(Object(Drupal\jsonapi\ResourceType\ResourceType), '/jsonapi')
#5 [internal function]: Drupal\jsonapi\Routing\Routes->routes()
#6 /var/www/html/drupal/web/core/lib/Drupal/Core/Routing/RouteBuilder.php(146): call_user_func(Array)
#7 /var/www/html/drupal/web/core/lib/Drupal/Core/ProxyClass/Routing/RouteBuilder.php(83): Drupal\Core\Routing\RouteBuilder->rebuild()
#8 /var/www/html/drupal/web/core/lib/Drupal/Core/Extension/ModuleInstaller.php(322): Drupal\Core\ProxyClass\Routing\RouteBuilder->rebuild()
#9 /var/www/html/drupal/web/core/lib/Drupal/Core/ProxyClass/Extension/ModuleInstaller.php(83): Drupal\Core\Extension\ModuleInstaller->install(Array, true)
#10 /var/www/html/drupal/vendor/drush/drush/src/Drupal/Commands/pm/PmCommands.php(79): Drupal\Core\ProxyClass\Extension\ModuleInstaller->install(Array, true)
#11 [internal function]: Drush\Drupal\Commands\pm\PmCommands->enable(Array, Array)
#12 /var/www/html/drupal/vendor/consolidation/annotated-command/src/CommandProcessor.php(257): call_user_func_array(Array, Array)
#13 /var/www/html/drupal/vendor/consolidation/annotated-command/src/CommandProcessor.php(212): Consolidation\AnnotatedCommand\CommandProcessor->runCommandCallback(Array, Object(Consolidation\AnnotatedCommand\CommandData))
#14 /var/www/html/drupal/vendor/consolidation/annotated-command/src/CommandProcessor.php(178): Consolidation\AnnotatedCommand\CommandProcessor->validateRunAndAlter(Array, Array, Object(Consolidation\AnnotatedCommand\CommandData))
#15 /var/www/html/drupal/vendor/consolidation/annotated-command/src/AnnotatedCommand.php(302): Consolidation\AnnotatedCommand\CommandProcessor->process(Object(Symfony\Component\Console\Output\ConsoleOutput), Array, Array, Object(Consolidation\AnnotatedCommand\CommandData))
#16 /var/www/html/drupal/vendor/symfony/console/Command/Command.php(255): Consolidation\AnnotatedCommand\AnnotatedCommand->execute(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/drupal/vendor/symfony/console/Application.php(971): Symfony\Component\Console\Command\Command->run(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/drupal/vendor/symfony/console/Application.php(248): Symfony\Component\Console\Application->doRunCommand(Object(Consolidation\AnnotatedCommand\AnnotatedCommand), Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/drupal/vendor/symfony/console/Application.php(148): Symfony\Component\Console\Application->doRun(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/drupal/vendor/drush/drush/src/Runtime/Runtime.php(112): Symfony\Component\Console\Application->run(Object(Drush\Symfony\DrushArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/drupal/vendor/drush/drush/src/Runtime/Runtime.php(41): Drush\Runtime\Runtime->doRun(Array)
#22 /var/www/html/drupal/vendor/drush/drush/drush.php(66): Drush\Runtime\Runtime->run(Array)
#23 /var/www/html/drupal/vendor/drush/drush/includes/preflight.inc(17): require('/var/www/html/d...')
#24 phar:///usr/local/bin/drush/bin/drush.php(141): drush_main()
#25 /usr/local/bin/drush(10): require('phar:///usr/loc...')
#26 {main}
TypeError: Argument 2 passed to Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}() must be an instance of Drupal\jsonapi\ResourceType\ResourceType, null given in Drupal\jsonapi\Routing\Routes::Drupal\jsonapi\Routing\{closure}() (line 435 of /var/www/html/drupal/web/modules/contrib/jsonapi/src/Routing/Routes.php).
 [error]  Drush command terminated abnormally due to an unrecoverable error. 
```

- `drush pmu jsonapi` to uninstall jsonapi because it's in a half baked state
- Pull in this PR to `controlled_access_terms`
- `drush -y fim controlled_access_terms_default_configuration`
- `drush en jsonapi`
- No explosions
